compiler/aarch64/codegen/ARM64SystemLinkage.cpp

```diff
@@ -847,3 +847,15 @@ TR::Register *TR::ARM64SystemLinkage::buildIndirectDispatch(TR::Node *callNode)
    TR_UNIMPLEMENTED();
    return NULL;
    }
+
+
+intptrj_t TR::ARM64SystemLinkage::entryPointFromCompiledMethod()
+   {
+   return (intptrj_t)(cg()->getCodeStart());
```

Comment:
I always feel a bit uneasy when I see a c-style cast like this because it makes it harder to reason about safety. Is this supposed to do a `reinterpret_cast`? If so, could we make this an explicit `reinterpret_cast`?

Reply:
Sure, I can fix that.